Task11/Task11/Program.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+namespace Table
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите число строк таблицы");
+            var m = int.Parse(Console.ReadLine());
+            Console.WriteLine("Введите число столбцов таблицы");
```

Comment:
Хотелось бы более понятное сообщение
```suggestion
            Console.WriteLine("Введите число строк таблицы (от 5 до 20)");
            var m = int.Parse(Console.ReadLine());
            Console.WriteLine("Введите число столбцов таблицы (от 5 до 20)");
```